joomla/joomla-cms
# Maintenance submenu

Pull Request for Improvement.
#### Summary of Changes

This simple PR aggregates com_checkin and com_cache menu option in a "Maintenance" submenu,

Note the titles of the views already follow that logic.

![image](https://cloud.githubusercontent.com/assets/9630530/14412439/c359b5fa-ff5a-11e5-97b3-66035ace52e4.png)

![image](https://cloud.githubusercontent.com/assets/9630530/14412440/c6da9988-ff5a-11e5-9389-5c594d22e2e4.png)

![image](https://cloud.githubusercontent.com/assets/9630530/14412442/c994081c-ff5a-11e5-9305-a1dad37292ce.png)
##### Before PR

![image](https://cloud.githubusercontent.com/assets/9630530/14412426/7f27d24a-ff5a-11e5-866e-487d97587098.png)
##### After PR

![image](https://cloud.githubusercontent.com/assets/9630530/14412427/83de283e-ff5a-11e5-9ff8-1be9e8fda844.png)
#### Testing Instructions
1. Use latest staging
2. Apply patch
3. Check the "System" menu
